src/Microsoft.ML.ImageAnalytics/MLImage.cs

```diff
@@ -126,6 +128,47 @@ private set
             }
         }
 
+        public ReadOnlySpan<byte> PixelsNoAlpha
```

Comment:
Suggestion: RGBPixels

Reply:
Would it make sense to make this a method?

ReadOnlySpan<byte> GetPixels(ColorFormat...)

Reply:
I made up ColorFormat  (or if it already exist I didn't necessarily mean that one :). Just seems like we might want

GetPixels(ColorFormat.RGB4)

GetPixels(ColorFormat.RGB8)

GetPixels(ColorFormat.ARGB4)

GetPixels(ColorFormat.ARGB8)


Maybe just a subset of https://learn.microsoft.com/en-us/dotnet/api/skiasharp.skcolortype?view=skiasharp-2.88 that we'd want to support.

Reply:
In our case yes it does always assume this order. But I think since this is so specific to our case I am going to move it from the MLImage class internal to the object detection code.

Reply:
@tarekgh Actually, the SKBitmap is private to the MLImage so I can't access it outside the class, and I need to access it. So how about I change the method name for now to just get RBG pixels, and then if we need to make it more advanced in the future we can. Cause we do already have a pixel extractor that is usually used, just in this case I can't do the extraction ahead of time so we are kinda a special case.

Reply:
My understanding is you need to filter and alpha channel anyway. Why don't you have a helper/extension method outside MLImage? It is simple to call `MLImage.Pixels` and then filter and reorder the pixels according to `MLIMage.PixelFormat`.